src/app/book/book.html

```diff
@@ -75,7 +75,7 @@
 			<input type="number" name="promo" placeholder="Enter Promo Code">
 		</div>
 		<div class="form-group">
-			<input type="submit" name="submit" value="Submit">
+			<input type="submit" name="submit" value="Submit" a="http://www.burgermonster.net/book/success">
```

Comment:
what is a="" ?

maybe you mean something different

try adding a ng-submit="success.submit" to the <form> tag
and in your controller creating a function call submit
inside the submit function you can call window.open(url)
this will redirect to the success page you wanted

make sure you set the function to a scope variable  
function submit () {
  window.open("http://www.burgermonster.net/book/success");
}
this.submit = submit;